### ISSUE TITLE
[usdview] slider jitters

### ISSUE THREAD

**Issue by AlexSchwank** [PixarAnimationStudios/USD#837]
### Description of Issue
When scrubbing the frame slider in usdview, it jitters and jumps between frame values. 

Printing values in _setValueFromEvent_ in [frameSlider.py](https://github.com/PixarAnimationStudios/USD/blob/dev/pxr/usdImaging/lib/usdviewq/frameSlider.py) you can see that the slider is set to 0,1,0,1,0,1,... when slowly scrubbing from frame 0 to frame 1.
```
[setValueFromEvent] currentValue =  0
[setValueFromEvent] movePosition =  0.0808080808081
[setValueFromEvent] targetPosition =  0.0
[setValueFromEvent] setting slider to =  1

[setValueFromEvent] currentValue =  1
[setValueFromEvent] movePosition =  0.0808080808081
[setValueFromEvent] targetPosition =  0.0
[setValueFromEvent] setting slider to =  0.0

[setValueFromEvent] currentValue =  0
[setValueFromEvent] movePosition =  0.0909090909091
[setValueFromEvent] targetPosition =  0.0
[setValueFromEvent] setting slider to =  1

[setValueFromEvent] currentValue =  1
[setValueFromEvent] movePosition =  0.10101010101
[setValueFromEvent] targetPosition =  0.0
[setValueFromEvent] setting slider to =  0.0

[setValueFromEvent] currentValue =  0
[setValueFromEvent] movePosition =  0.111111111111
[setValueFromEvent] targetPosition =  0.0
[setValueFromEvent] setting slider to =  1
```
This due to the _targetPosition_ being increased or decreased according to a _movePosition_. Though I'm not sure what is the intention behind this?

Taking out the following code
```
         if targetPosition == currentValue:
             if (movePositTion - currentValue) >= 0:
                 targetPosition = currentValue + 1
             else:
                 targetPosition = currentValue - 1;
```
leads to smooth scrubbing and these outputs:
```
[setValueFromEvent] currentValue =  0
[setValueFromEvent] movePosition =  0.10101010101
[setValueFromEvent] targetPosition =  0.0
[setValueFromEvent] setting slider to =  0.0

[setValueFromEvent] currentValue =  0
[setValueFromEvent] movePosition =  0.111111111111
[setValueFromEvent] targetPosition =  0.0
[setValueFromEvent] setting slider to =  0.0
//...
[setValueFromEvent] currentValue =  0
[setValueFromEvent] movePosition =  0.494949494949
[setValueFromEvent] targetPosition =  0.0
[setValueFromEvent] setting slider to =  0.0

[setValueFromEvent] currentValue =  0
[setValueFromEvent] movePosition =  0.505050505051
[setValueFromEvent] targetPosition =  1.0
[setValueFromEvent] setting slider to =  1.0
```

Though I wonder if we're the only ones running into this issue?

### Steps to Reproduce
1. usdview /path/to/animatedAsset.usd
2. Scrub the frame slider

### Package Versions
19.01+

**Comment by spiffmon** [comment]
Thanks to @ianlawson, this was fixed in 19.05. Closing!

**Comment by AlexSchwank** [comment]
Hi @spiffmon, do you mean PR #770 or is there another PR because that one only got merged into dev 4 days ago...

**Comment by spiffmon** [comment]
There were a couple of earlier fixes that made it into 19.05, but I apologize I don’t recall which one fully fixed the jittering, so it could indeed be the most recent one that is only in dev, Alex.

**Comment by AlexSchwank** [comment]
After fixing some internal GLEW issues I was able to test the current dev branch and it all works :) 
Thanks @spiffmon and @ianlawson !
